### DISCUSSION TITLE
Build on doc changes

### DISCUSSION THREAD

**Discussion by jason-capsule42** [AlaskaAirlines/WC-Generator#510] [Announcements]
Now that the AuroDocSite is changing to use components directly and to render documentation specific the version that is running on the docsite, we need to start executing builds when do "doc(...): ...." commits.

Thoughts?

**Comment by blackfalcon** [comment]
It is not reasonable to think that we can release a new build of the doc site with every new release of a component. That being said, we DO need to update the doc site with any minor/feature releases. E.g. `v1.4.0 => v1.5.0` This is to ensure that the version installed in the site is compatible with the newly updated documentation to support that feature. 

Outside of that, I would assume that it is appropriate to do a general update of all Auro dependencies once a sprint to keep things up to date.
